scripts/README.md

```diff
@@ -0,0 +1,157 @@
+# Scripts
+
+This is a set of boilerplate scripts describing the [normalized script pattern that GitHub uses in its projects](https://github.blog/2015-06-30-scripts-to-rule-them-all/). The [GitHub Scripts To Rule Them All
+](https://github.com/github/scripts-to-rule-them-all) was used as a template. They were tested using Ubuntu 18.04.3 LTS on Windows 10 and using GridOS 26, a derivative of Red Hat Fedora 26, on the Lincoln Laboratory Supercomputing Cluster.
+
+- [Scripts](#scripts)
+  - [`FLOOD_ANALYSIS_CORE` and Execution](#flood_analysis_core-and-execution)
+  - [Dependencies](#dependencies)
+    - [Linux Shell](#linux-shell)
+    - [Proxy and Internet Access](#proxy-and-internet-access)
+    - [Superuser Access](#superuser-access)
+  - [The Scripts](#the-scripts)
+    - [script/bootstrap](#scriptbootstrap)
+      - [Packages](#packages)
+      - [Git Submodules](#git-submodules)
+    - [script/setup](#scriptsetup)
+      - [Data](#data)
+
+## `FLOOD_ANALYSIS_CORE` and Execution
+
+These scripts assume that `FLOOD_ANALYSIS_CORE` has been set. Refer to the repository root [README](../README.md) for instructions.
```

Comment:
There are no such instructions in the repository root README.

Reply:
I'll address this

Reply:
This should had been addressed by commit 16e329b